.eslintrc

```diff
@@ -42,7 +44,10 @@
       "files": [
         "core/tests/protractor/*.js",
         "core/tests/protractor_utils/*.js",
-        "core/tests/protractor_desktop/*.js"
+        "core/tests/protractor_desktop/*.js",
+        "core/tests/webdriverio/*.js",
+        "core/tests/webdriverio_utils/*.js",
+        "core/tests/webdriverio_desktop/*.js"
```

Comment:
Why are we adding these files here? (Considering we are expected to fix the issues in the listed files here)

Reply:
We are planning to keep the same lint files for both the version, that's why I have added these files here instead of creating a new custom eslint rule.
@SAEb-ai PTAL.
Thanks

Reply:
Looks like I misunderstood the changes, these files listed here are "included" and "not excluded", sorry for the misunderstanding!